tests/sweep_framework/sweeps/eltwise/unary/lgamma/lgamma_sharded.py

```diff
@@ -29,7 +35,7 @@
 # Developers can create their own generator functions and pass them to the parameters as inputs.
 parameters = {
     "nightly": {
-        "input_spec": gen_sharded_spec_unary(16, max_tensor_size_per_core=20 * 1024, layouts=["TILE_LAYOUT"]),
```

Comment:
max_tensor_size_per_core=20 * 1024 was intentional because lgamma works slower.

Reply:
I will return it.